apps/coscrad-frontend/src/components/login-button/login-button.tsx

```diff
@@ -1,18 +1,20 @@
 import { useAuth0 } from '@auth0/auth0-react';
 import LoginIcon from '@mui/icons-material/Login';
-import { IconButton } from '@mui/material';
+import { IconButton, Tooltip } from '@mui/material';
 
 const LoginButton = () => {
     const { loginWithRedirect } = useAuth0();
 
     return (
-        <IconButton
-            data-testid="login-button"
-            color="secondary"
-            onClick={() => loginWithRedirect()}
-        >
-            <LoginIcon />
-        </IconButton>
+        <Tooltip title="Login">
```

Comment:
Isn't it "Log In" when used as a verb and "Login" when used as a noun? Which do we want here? I always have a hard time with this.

Reply:
Just habit...  Will update!